locale/en/blog/uncategorized/10-lts-to-12-lts.md

```diff
@@ -0,0 +1,5039 @@
+With the transition of Node.js 12 from LTS into Current, a suite of new features have been added to Node.js!
+
+In each release's changelog we capture notable changes – but going through over two dozen individual release notes to find those can be a bit daunting.
+
+As such, I've pulled together **all release notes** from every release from Node.js 11.0.0 to 12.12.0 and included an even higher level "Best of the Best: Most Notable Changes" section that allows you to see what some of the biggest changes are between Node.js v10.x LTS at launch and Node.js v12.x LTS at launch.
+
+## Best of the Best: Most Notable Changes
+
+This section is a highlight of the most notable changes, as selected from all "[Notable Changes](#notable-changess)" from every release since Node.js v10 went LTS.
+
+* **assert**:
+  * Legacy mode deprecation (`DEP0089`) is revoked (Colin Ihrig) [#28892](https://github.com/nodejs/node/pull/28892)
+* **build**:
+  * FreeBSD 10 is no longer supported. [#22617](https://github.com/nodejs/node/pull/22617)
+* **compression / zlib**:
+  * Added brotli support (Anna Henningsen and Zach Vacura) [#24938](https://github.com/nodejs/node/pull/24938)
+* **deps**:
+  * libuv
+    * Now shipping libuv 1.31.0, and all previous updates since the latest version of libuv that shipped in Node.js v10 LTS.
+  * npm
+    * Now shipping npm 6.10.3, and all previous updates since the latest version of npm that shipped in Node.js v10 LTS.
+    * For more details, see the [npm releases' changelog](https://github.com/npm/cli/releases).
+  * OpenSSL
+    * Now shipping OpenSSL 1.1.1c, and all previous updates since the latest version of OpenSSL that shipped in Node.js v10 LTS.
+    * For more details, see the [OpenSSL releases' changelog](https://www.openssl.org/news/changelog.html)
+  * V8
+    * Now shipping V8 7.7.299.11, and all previous updates since the latest version of V8 that shipped in Node.js v10 LTS.
```

Comment:
V8 7.7.299.13 since Node.js 12.12.0.

Reply:
And again:

```suggestion
    * Now shipping V8 7.7.299.13, and all previous updates since the latest version of V8 that shipped in Node.js v10 LTS.
```